website/documentation/guides/monitoring_and_troubleshooting/analysing-node-failures/_index.md

```diff
@@ -0,0 +1,68 @@
+
+A potential result might reveal
+```
+{"_entry":"Node ip-10-55-138-185.eu-central-1.compute.internal status is now: NodeNotReady","count":1,"firstTimestamp":"2023-04-05T12:02:08Z","lastTimestamp":"2023-04-05T12:02:08Z","namespace":"default","object":"Node/ip-10-55-138-185.eu-central-1.compute.internal","origin":"shoot","reason":"NodeNotReady","source":"node-controller","type":"Normal"}
+```
+
+### Check machine-controller-manager Logs
+If a node was getting unhealthy, the last conditions can be found in the logs of the `machine-controller-manager` by using a query like `{pod_name=~"machine-controller-manager.*"}|="<node-name>"`.
+
+**Caveat**: every `node` resource is backed by a corresponding `machine` resource managed by machine-controller-manager. Usually two corresponding `node` and `machine` resources have the same name with the exception of AWS. Here you first need to find with the above query the corresponding `machine` name, typically via a log like this
+```
+2023-04-05 12:02:08	{"log":"Conditions of Machine \"shoot--demo--cluster-pool-z1-6dffc-jh4z4\" with providerID \"aws:///eu-central-1/i-0a6ad1ca4c2e615dc\" and backing node \"ip-10-55-138-185.eu-central-1.compute.internal\" are changing","pid":"1","severity":"INFO","source":"machine_util.go:629"}
+```
+This reveals that `node` `ip-10-55-138-185.eu-central-1.compute.internal` is backed by `machine` `shoot--demo--cluster-pool-z1-6dffc-jh4z4`. On infrastructures other than AWS you can omit this step.
+
+With the machine name at hand, now search for log entries with `{pod_name=~"machine-controller-manager.*"}|="<machine-name>"`.
+In case the node had failing conditions, you'd find logs like this:
+```
+2023-04-05 12:02:08	{"log":"Machine shoot--demo--cluster-pool-z1-6dffc-jh4z4 is unhealthy - changing MachineState to Unknown. Node conditions: [{Type:ClusterNetworkProblem Status:False LastHeartbeatTime:2023-04-05 11:58:39 +0000 UTC LastTransitionTime:2023-03-23 11:59:29 +0000 UTC Reason:NoNetworkProblems Message:no cluster network problems} ... {Type:Ready Status:Unknown LastHeartbeatTime:2023-04-05 11:55:27 +0000 UTC LastTransitionTime:2023-04-05 12:02:07 +0000 UTC Reason:NodeStatusUnknown Message:Kubelet stopped posting node status.}]","pid":"1","severity":"WARN","source":"machine_util.go:637"}
+```
+
+In the example above, the reason for an unhealthy node was that `kubelet` failed to renew its heartbeat.
+Whatever the failure condition might have been, if a node gets unhealthy, it will be terminated by `machine-controller-manager` after the `machineHealthTimeout` has elapsed (this parameter can be configured in your [shoot spec](https://github.com/gardener/gardener/blob/v1.68.0/example/90-shoot.yaml#L132))
```

Comment:
```suggestion
Whatever the failure condition might have been, if a node gets unhealthy, it will be terminated by `machine-controller-manager` after the `machineHealthTimeout` has elapsed (this parameter can be configured in your [shoot spec](https://github.com/gardener/gardener/blob/v1.68.0/example/90-shoot.yaml#L132)).
```